miyagi_trainer/train.py

```diff
@@ -146,6 +148,21 @@ def train_model(model,
             epoch_acc = 100 * running_corrects.double() / dataset_sizes[phase]
             print(f'{phase} Loss: {epoch_loss:.4f} Acc: {epoch_acc:.2f}%')
 
+            if save_model:
+                save_curr_model = (not metric_eer and epoch_acc > best_acc) or \
+                                  (metric_eer and epoch_eer < best_eer)
+                if save_curr_model:
+                    model_folder = wandb.run.name if track_experiment else \
+                                   datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+                    if not os.path.exists(model_folder):
+                        os.mkdir(model_folder)
+                    torch.save({
+                            'epoch': epoch,
+                            'model_state_dict': model.state_dict(),
+                            'optimizer_state_dict': optimizer.state_dict(),
+                            'loss': loss,
+                            }, os.path.join(model_folder, "best_model_save.pth"))
```

Comment:
I will call by default `best_saved_model.pth` and whats the best extension here `.pth` or `pt`?

Reply:
It appears that `.pt` is preferable, according to https://discuss.pytorch.org/t/what-does-pth-tar-extension-mean/36697/3. 
I will modify it.